tests/e2e/e2e_util_test.go

```diff
@@ -184,32 +231,85 @@ func (s *IntegrationTestSuite) withdrawReward(c *chain, valIdx int, endpoint, pa
 	)
 }
 
-func (s *IntegrationTestSuite) sendIBC(srcChainID, dstChainID, recipient string, token sdk.Coin) {
+//
+//func (s *IntegrationTestSuite) sendIBC(srcChainID, dstChainID, recipient string, token sdk.Coin) {
+//	ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
+//	defer cancel()
+//
+//	s.T().Logf("sending %s from %s to %s (%s)", token, srcChainID, dstChainID, recipient)
+//
+//	exec, err := s.dkrPool.Client.CreateExec(docker.CreateExecOptions{
+//		Context:      ctx,
+//		AttachStdout: true,
+//		AttachStderr: true,
+//		Container:    s.hermesResource.Container.ID,
+//		User:         "root",
+//		Cmd: []string{
+//			"hermes",
+//			"tx",
+//			"raw",
+//			"ft-transfer",
+//			dstChainID,
+//			srcChainID,
+//			"transfer",  // source chain port ID
+//			"channel-0", // since only one connection/channel exists, assume 0
+//			token.Amount.String(),
+//			fmt.Sprintf("--denom=%s", token.Denom),
+//			fmt.Sprintf("--receiver=%s", recipient),
+//			"--timeout-height-offset=1000",
+//		},
+//	})
+//	s.Require().NoError(err)
+//
+//	var (
+//		outBuf bytes.Buffer
+//		errBuf bytes.Buffer
+//	)
+//
+//	err = s.dkrPool.Client.StartExec(exec.ID, docker.StartExecOptions{
+//		Context:      ctx,
+//		Detach:       false,
+//		OutputStream: &outBuf,
+//		ErrorStream:  &errBuf,
+//	})
+//	s.Require().NoErrorf(
+//		err,
+//		"failed to send IBC tokens; stdout: %s, stderr: %s", outBuf.String(), errBuf.String(),
+//	)
+//
+//	s.T().Log("successfully sent IBC tokens")
+//}
+
+func (s *IntegrationTestSuite) sendIBC(c *chain, valIdx int, sender, recipient, token, fees string) {
 	ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
 	defer cancel()
 
-	s.T().Logf("sending %s from %s to %s (%s)", token, srcChainID, dstChainID, recipient)
+	ibcCmd := []string{
```

Comment:
could we use `s.executeGaiaTxCommand(ctx, c, ibcCmd, valIdx, endpoint)` for this?

Reply:
yeah, we should refactor all the gaia tx cli into `s.executeGaiaTxCommand(ctx, c, ibcCmd, valIdx, endpoint)`